test/jest/Connext.spec.ts

```diff
@@ -105,25 +108,28 @@ describe('ConnextClient', () => {
             return { freeBalanceOnChain: 0 };
         }
       });
-      connext['sendRequest'] = jest.fn();
+      connext['ethProvider'] = ({
+        onChainTransfer,
+      } as unknown) as EthProvider;
     });
 
     afterEach(() => {
       jest.clearAllMocks();
-    }),
-      it('fails with custom fee', async () => {
-        expect.assertions(1);
-        try {
-          await connext.withdraw({
-            currency: 'ETH',
-            destination: DESTINATION_ADDRESS,
-            amount: 123,
-            fee: 1,
-          });
-        } catch (e) {
-          expect(e).toMatchSnapshot();
-        }
-      });
+    });
+
+    it('fails with custom fee', async () => {
```

Comment:
Why are we not allowing custom fees?

Reply:
Not implemented, yet. This is porting over the functionality that we had for V1 to V2. We'll of course support this in the future.

Reply:
Created an issue for this https://github.com/ExchangeUnion/xud/issues/2046